workflow-config/config-android/src/main/java/com/squareup/workflow1/config/AndroidRuntimeConfigTools.kt

```diff
@@ -22,12 +22,21 @@ public class AndroidRuntimeConfigTools {
      *      to the UI layer.
      * "baseline" : [RenderPerAction] Original Workflow Runtime. Note that this doesn't need to
      *      be specified as it is the current default and is assumed by this utility.
+     *
+     * Then, these can be combined (via '-') with:
+     * "stateChange" : Only re-render when the state of some WorkflowNode has been changed by an
+     *   action cascade.
+     *
+     * E.g., "baseline-stateChange" to turn on the stateChange option with the baseline runtime.
+     *
      */
     @WorkflowExperimentalRuntime
     public fun getAppWorkflowRuntimeConfig(): RuntimeConfig {
       return when (BuildConfig.WORKFLOW_RUNTIME) {
-        "conflate" -> ConflateStaleRenderings
-        else -> RenderPerAction
+        "conflate" -> ConflateStaleRenderings(renderOnStateChangeOnly = false)
+        "conflate-stateChange" -> ConflateStaleRenderings(renderOnStateChangeOnly = true)
+        "baseline-stateChange" -> RenderPerAction(renderOnStateChangeOnly = true)
+        else -> RuntimeConfig.DEFAULT_CONFIG
```

Comment:
Seems like we should be a bit more strict so that we catch typos for people.

```suggestion
        "conflate" -> ConflateStaleRenderings(renderOnStateChangeOnly = false)
        "conflate-stateChange" -> ConflateStaleRenderings(renderOnStateChangeOnly = true)
        "baseline-stateChange" -> RenderPerAction(renderOnStateChangeOnly = true)
        "", "baseline" -> RuntimeConfig.DEFAULT_CONFIG
        else -> throw IllegalArgumentException("Unrecognized config \"${BuildConfig.WORKFLOW_RUNTIME}\"")
```

Reply:
I like the Set<Enum> idea!

I'm going to leave it as strings for the options though as I don't want to mess with the buildConfig stuff I set up before (the command line options are easy - but the buildconfig for the instrumentation tests is harder IIRC)